public/app/features/alerting/unified/api/alertmanagerApi.ts

```diff
@@ -103,10 +104,16 @@ export const alertmanagerApi = alertingApi.injectEndpoints({
     getExternalAlertmanagers: build.query<ExternalAlertmanagers, void>({
       query: () => ({ url: '/api/v1/ngalert/alertmanagers' }),
       transformResponse: (response: ExternalAlertmanagersResponse) => response.data,
+      providesTags: ['AlertmanagerConnectionStatus'],
     }),
 
     saveExternalAlertmanagersConfig: build.mutation<{ message: string }, ExternalAlertmanagerConfig>({
-      query: (config) => ({ url: '/api/v1/ngalert/admin_config', method: 'POST', data: config }),
+      query: (config) => ({
+        url: '/api/v1/ngalert/admin_config',
+        method: 'POST',
+        data: config,
+        showSuccessAlert: false,
+      }),
       invalidatesTags: ['AlertmanagerChoice'],
```

Comment:
Do you need to/is it worth invalidating `AlertmanagerConnectionStatus` here as well?

Reply:
Hm it shouldn't matter too much since the FE will just ignore any Alertmanager URLs that can't be matched to configured Alertmanager data sources, but it does make sense from a semantic point of view to mark the connection status as stale when we make modifications to AM data sources :D